<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sex:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BMI:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Children:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Smoker:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Region:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget::item:selected
{
    background: 
	color: rgb(85, 170, 255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>Linear Regression</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polynomal Regression</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random Forest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decision Tree</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { color: #000000; border: 2px solid #555; border-radius: 20px; border-style: outset; background: qradialgradient( cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 5, stop: 0 #7ddcff, stop: 1 #888 ); padding: 5px; }
color: rgb(125, 220, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>Schätzen</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Schätzung von medizinischem Aufwand</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Result">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_about">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: double; border-width: 4px; border-color: rgb(125, 220, 255); border-radius: 4px; 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_region">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox { border: 1px solid gray; border-radius: 3px; padding: 1px 18px 1px 3px; min-width: 6em; }</string>
   </property>
   <item>
    <property name="text">
     <string>Northeast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Southeast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Southwest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Northwest</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_smoker">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox { border: 1px solid gray; border-radius: 3px; padding: 1px 18px 1px 3px; min-width: 6em; }</string>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_sex">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox { border: 1px solid gray; border-radius: 3px; padding: 1px 18px 1px 3px; min-width: 6em; }</string>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_modell_auswahl">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Wählen Sie ein Modell aus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_parameter">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Geben Sie den Inputparametern ein</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INF 523 Projekt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
